concepts/translations/base-language-selection.md

```diff
@@ -0,0 +1,49 @@
+# Defining Base Languages in Shopware’s Region-Independent Snippet Layer
+
+With Shopware 6.7 a **region-independent snippet layer** was introduced to reduce duplicate translations.  
+In this model the snippet loader first attempts to load a region-specific variant (e.g. `de-DE`), then falls back to a **base language** (e.g. `de`), and finally falls back to `en` as a universal default.  
+The base layer concept and fallback order are explained in detail in the [Built-in Translation Handling](built-in-translation-system.html) page, which you should read alongside this guide.
+
+## Why a base layer?
+
+Before v6.7, Shopware shipped only region-specific snippet files. Developers often duplicated an existing file (for example `en-GB`) and changed a handful of keys for another locale.  
+This practice led to bloated repositories and inconsistent fallbacks. The **region-independent layer** centralizes common translations into a neutral base file (`en`, `de`, `pt`, etc.) and isolates regional differences in small patch files.  
+Detailed examples of how Shopware resolves translations are available in the [Built-in Translation Handling](built-in-translation-system.html) guide.
+
+## Criteria for selecting a base language
+
+A base language is a **language code without a regional sub-tag** (for example `en`, `de` or `pt`).  
+Shopware’s core repository defines which languages have a base file; if your language appears only as a single locale (such as `sv-SE`), that file serves as both the base and regional variant.  
+To decide whether your language needs a base file and which variant defines it, follow these principles:
+
+* **Use the standard variant** – The base file should reflect the official standard form of a language. Shopware uses British English (en-GB) as the basis for `en` and European Portuguese (pt-PT) for `pt`.
+* **Create a base only when multiple regional variants exist** – If there is only one supported locale (for example Czech), you don’t need a separate `cs` file.
+* **Aim for neutrality** – Avoid region-specific terminology in the base file. For Spanish, a neutral Castilian register is recommended to maximize comprehension.
```

Comment:
What do you wanna say with this?
Our core uses agnostic files (We should not use "base" files, since messages.<locale>.base is something different) as a best practice. This sounds like it's best practice for swedish which isn't true. Of course, for languages only spoken at one place, it doesn't make a difference, but still the agnostic one should be the best practice.

I guess these lines only confuse people and can safely be removed. ("Aim for dialect neutrality if possible" can stay in some regards I guess)

Reply:
hmm.. didnt see this comment somehow. removed. the other points